app/controllers/waivers_controller.rb

```diff
@@ -11,7 +11,9 @@ def new
     if @user.has_signed_waiver
       flash[:notice] = "You have already agreed to the release."
     end
-    
+
+    @should_see_video = !current_user.participant.is_scc? or params[:participant_id] == nil
```

Comment:
`@user.is_scc?` should not check for `nil` participant, but I will change the first part